node/src/components/block_accumulator.rs

```diff
@@ -808,6 +811,9 @@ impl<REv: ReactorEvent> Component<REv> for BlockAccumulator {
             Event::ExecutedBlock { hot_block } => {
                 let height = hot_block.block.header().height();
                 let era_id = hot_block.block.header().era_id();
+                // Logically it makes sense to purge the acceptors here or in
+                // `register_local_tip`, but only on the path where a new tip
+                // is accepted.
                 self.register_local_tip(height, era_id);
```

Comment:
we do purge here (first line inside `register_local_tip`)

Reply:
Yes, that is what I meant there.